app/controllers/user/mygardens/plants_controller.rb

```diff
@@ -8,12 +8,16 @@ def create
 		plant = Plant.find(params[:plant_id])
     garden.plants << plant
     garden.save
-    add_notification(garden, plant)
+    Thread.new do
+      add_notification(garden, plant)
+    end
```

Comment:
added a thread to run this task in the background

Reply:
nice!